package.json

```
@@ -16,13 +16,16 @@
     "eslint": "^7.12.1",
     "eslint-config-prettier": "^6.15.0",
     "prettier": "^2.1.2",
+    "synckit": "^0.8.6",
```

```suggestion
    "synckit": "^0.8.8",
```

Let's make sure it's using the latest much slimmer version.

---

I already had that change locally and ready to push, I also wanted to add some other changes to the PR, including an up-to-date benchmark with my latest findings